build/changelog.py

```diff
@@ -63,13 +128,16 @@ def parse_args():
 def main():
 
     args = parse_args()
+    mention = ""
     changelog = {}
-
     for commit_id in get_commit_ids(args.from_version, args.to_commit):
         commit_message = get_commit_message(commit_id)
-        group, line = get_changelog_message(commit_message, args.repo_url)
+        if not is_maintainer(commit_message):
+            mention = mention_author(commit_message, commit_id, args.token)
+        group, line = get_changelog_message(commit_message, mention, args.repo_url)
         changelog.setdefault(group, []).append(line)
-
+        mention = ""
+    
```

Comment:
If you move `mention = ""` inside the loop, like right before line 135, you can then remove the extra one on line 139